protocol/shannon/fullnode_account_fetcher.go

```diff
@@ -1,6 +1,9 @@
 package shannon
 
-// TODO_TECHDEBT(@commoddity): Refactor (remove?) this whole file based on the details in #291
+// TODO_TECHDEBT(@commoddity): Refactor (remove?) this whole file
```

Comment:
Confirming this file is removed in the subsequent refactor PRs, specifically as part 1, which is #295:
https://github.com/buildwithgrove/path/pull/295